tests/console/yast2_i.pm

```diff
@@ -20,7 +20,7 @@
 # "nfs-client"
 # - Disable "recommended packages" and accept install of test package
 # - Remove test package
-# Maintainer: Martin Kravec <mkravec@suse.com>
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
```

Comment:
completely unnecessary but up to you... we already talked offline about it, and if you are not changing a lot this module keep the original writer (even if the mantainer field is useless) is fine.

Reply:
IMO it is essential. Martin is not part of the team and we should take full responsibility.

Reply:
we should not be touching headers/licenses of files that we don't modify extensively, the current maintainer might still be the best contact person and we might not know very well the code in this module. if someone or some other team reach us  with this we might still consider not schedule this module at all, or create a complete different module.  There is a lot of shared code in the repo, for that reason it doesn't make sense at all, and we should narrow our responsibility to what we know very well. Still if you want to do it, do it, no problem for me, it is a YaST module after all.

Reply:
Obviously i am not aware of many of the above. but this does not change that this is our responsibility either we know the module well or not.